components/infobox/wikis/starcraft2/infobox_person_player.lua

```diff
@@ -0,0 +1,556 @@
+local Player = require('Module:Infobox/Person')
+local Variables = require('Module:Variables')
+local Achievements = require('Module:Achievements in infoboxes')._player
+local RaceIcon = require('Module:RaceIcon').getBigIcon
+local String = require('Module:StringUtils')
+local Matches = require('Module:Upcoming ongoing and recent matches player/new')
+
+local _EPT_SEASON = 2021
+local _ALLKILLICON = '[[File:AllKillIcon.png|link=All-Kill Format]]&nbsp;×&nbsp;'
+
+local earningsGlobal = {}
+local pagename = mw.title.getCurrentTitle().prefixedText
+local currentYear = tonumber(os.date('%Y'))
+local shouldStoreData
+local hasAchievements
+local raceData
+local statusStore
+local militaryStore
+
+local StarCraft2Player = {}
```

Comment:
There was discussion about changing this to `CustomPlayer` over `SC2Player`

Reply:
did we reach a decision?
because atm all the modules still use the wikinames in them
(is a simple search & replace anyways)

Reply:
> Rathoz — Today at 15:27
> Should the Classes in /Custom be called Custom<BaseClass> or <wikiname><baseclass>?
> Vogan — Today at 15:48
> Probably the former

Let's finish it conversation tomorrow

Reply:
monday (vogan is not active on the weekend)^^

Reply:
My vote is `CustomPlayer`